src/js/editor/editor.js

```diff
@@ -565,6 +540,16 @@ class Editor {
     this.addCallback(CALLBACK_QUEUES.CURSOR_DID_CHANGE, callback);
   }
 
+  /**
+   * @method didReparse
+   * @param {Function} callback This callback is called after any part of the
+   *        post is reparsed
+   * @public
+   */
+  didReparse(callback) {
```

Comment:
This should also be documented in [the README](https://github.com/bustlelabs/mobiledoc-kit#editor-lifecycle-hooks)